From a0ab8e74ceca43dec30014d571b1da4522f20766 Mon Sep 17 00:00:00 2001
From: Josue David Hernandez Gutierrez <josue.d.hernandez.gutierrez@intel.com>
Date: Wed, 27 Feb 2019 16:42:33 -0600
Subject: [PATCH] CLR custimizations

hide the screenshot section if there is not screenshot available

Signed-off-by: Josue David Hernandez Gutierrez <josue.d.hernandez.gutierrez@intel.com>
---
 src/gs-details-page.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/gs-details-page.c b/src/gs-details-page.c
index 3e7d3196..ee3559ba 100644
--- a/src/gs-details-page.c
+++ b/src/gs-details-page.c
@@ -698,6 +698,7 @@ gs_details_page_refresh_screenshots (GsDetailsPage *self)
 				screenshots->len > 0);
 	if (screenshots->len == 0) {
 		gs_container_remove_all (GTK_CONTAINER (self->box_details_screenshot_thumbnails));
+		gtk_widget_set_visible (self->box_details_screenshot_fallback, FALSE);
 		return;
 	}
 
-- 
2.20.1

